flow/table.go

```diff
@@ -493,8 +755,8 @@ func (ft *Table) Run() {
 			}
 		case ps := <-ft.packetSeqChan:
 			ft.processPacketSeq(ps)
-		case op := <-ft.flowChanOperation:
-			ft.processFlowOP(op)
+		case ebpfFlow := <-ft.flowEBPFChan:
```

Comment:
please do not remove the flowOpChan as it will be used for other probes

Reply:
done